jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractScreenListPresenter.java

```diff
@@ -71,17 +70,20 @@ public void setServerTemplateSelectorMenuBuilder(final ServerTemplateSelectorMen
     }
 
     public void onServerTemplateSelected(@Observes final ServerTemplateSelected serverTemplateSelected) {
-        setSelectedServerTemplate(serverTemplateSelected.getServerTemplateId());
+        setSelectedServerTemplate(serverTemplateSelected.getServerTemplateId(), false);
     }
 
     public String getSelectedServerTemplate() {
         return selectedServerTemplate;
     }
 
-    protected void setSelectedServerTemplate(final String selectedServerTemplate) {
+    protected void setSelectedServerTemplate(final String selectedServerTemplate, final Boolean forceRefresh) {
```

Comment:
Demo seemed successful :)
One comment I have at this point is that this two argument setter looks more as a workaround then the way to go when we start redesigning, but I have no suggestions at the moment.

Reply:
I agree it isn't the nicest change but... as you said, there will be probably more changes related to this once we actually get the new designs in.